### ISSUE TITLE
[Feature Request] Open in existing Windows Terminal instance

### ISSUE THREAD

**Issue by ZeeWanderer** [desktop/desktop#9949]
If Windows Terminal is selected as a shell opening repo with it from GUI results in a new Terminal window. This might be desirable in some cases but multiple shells/tabs in a single window is the Windows Terminal thing.
Would it be possible to add an option to open repo in the last active Terminal instance?

EDIT:
The feature to add tabs to existing instance is being tracked on Windows Terminal side.

link microsoft/terminal#4472
link microsoft/terminal#4632

**Comment by rafeca** [comment]
Thanks for the suggestion @ZeeWanderer, and thanks for linking to the appropiate issues on Windows Terminal.

We're going to monitor microsoft/terminal#4472 to see how that's implemented on the terminal side and see what's the best way to integrate with that feature once it becomes available.

Something worth to note is that we try to avoid adding new configuration options whenever possible, so depending on the implementation on the Microsoft Terminal side, we'll have to discuss whether we want to change the default behaviour when opening that terminal or not.

**Comment by just1a-person** [comment]
This already works. This issue can be closed

**Comment by ZeeWanderer** [comment]
@itsme-alan Nope, it does not. Opening in existing windows works for `wt` from command line but GitHub desktop provides no such function. `Open in Windows Terminal` always opens a new window and maybe i am mistaken but i see no options to make it use existing terminal window. So, unless developers indicate that this feature will not be added at all i prefer to keep this issue open.

**Comment by just1a-person** [comment]
Maybe this feature is only available in Windows Terminal Preview. Also Terminal's settings has to configured this way:

![Windows Terminal instance settings](https://user-images.githubusercontent.com/75578443/122319399-363fb980-cf3e-11eb-9758-252913bcba58.png)

**Comment by ZeeWanderer** [comment]
@itsme-alan Thank you for showing me that. I guess i can close the issue but that still is not a Github Desktop feature and would force that behaviour on system level which is not ideal. Thou i doubt it will be implemented on Github Desktop level anyway...

**Comment by just1a-person** [comment]
I think a better approach would be to allow users to add custom arguments to `wt.exe` in Settings.


Also Windows Terminal might end up being completely removed from options: see #12463
